### PR TITLE
Improve codegen quality (shorter typenames)

### PR DESCRIPTION
**Pull request by pfrazee**
- Update lex-cli to produce shorter typenames once again
- Update all generated code to use these shorter names

### CODE REVIEW

File: packages/server/src/api/app/bsky/util/feed.ts [hunk 1]
```diff
@@ -35,8 +35,7 @@ export enum FeedAlgorithm {
   ReverseChronological = 'reverse-chronological',
 }
 
-type FeedItem = GetAuthorFeed.AppBskyGetAuthorFeedFeedItem &
-  GetHomeFeed.AppBskyGetHomeFeedFeedItem
+type FeedItem = GetAuthorFeed.FeedItem & GetHomeFeed.FeedItem
```
Comment (devinivy) [inline packages/server/src/api/app/bsky/util/feed.ts:38]: 💎 this looks so much nicer and more intuitive.

**Review by devinivy — APPROVED** [review]
Big yep, looks good!  This will be a nice life improvement.

File: packages/server/src/api/app/bsky/util/feed.ts (hunk 1 above)
Comment (pfrazee) [reply packages/server/src/api/app/bsky/util/feed.ts]: Yeah, really clutch

**Review by dholms — APPROVED** [review]
yeah this is niicee 👌

**Pull request merged**